test/config.py

```diff
@@ -6,7 +6,7 @@
 
     'host': 'localhost',
     'port': 9000,
-    'url_base': '/irods-http-api/0.4.0',
+    'url_base': '/irods-http-api/<version>',
```

Comment:
Don't forget to update the tests to populate the version number through some means. Otherwise, we won't be able to test the endpoints.

Reply:
Not sure what you mean. It is the dev's responsibility to properly configure the test config file before launching the tests. Having the build system update the version was a convenience, not a requirement.